pubspec.yaml

```diff
@@ -53,16 +53,17 @@ dependencies:
   shared_preferences: ^2.2.2
   styled_text: ^8.1.0
   url_launcher: ^6.1.14
-  walletconnect_flutter_v2: 2.2.1
+  walletconnect_flutter_v2: ^2.2.3
   web3dart: ^2.7.1
+  yaml_edit: ^2.2.1
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
   build_runner: any
   flutter_launcher_icons: ^0.13.1
-  flutter_lints: ^3.0.1
-  isar_generator: *isar_version
+  flutter_lints: ^4.0.0
+  isar_generator: 3.1.0+1
```

Comment:
`yaml_edit` library was not supporting aliases. Upgraded versions, but I'm light in `flutter`'s internals. Not sure if the yaml edit library is internal to flutter instead of project related. 

Let me know if you have any ideas.

Reply:
```
Encountered an alias node along [dependencies, isar]! Alias nodes are nodes that refer to a previously serialized nodes, and are denoted by either the "*" or the "&" indicators in the original YAML. As the resulting behavior of mutations on these nodes is not well-defined, the operation will not be supported by this library.
```

Reply:
Reverted to flutter version 3.16 and updated the files